### ISSUE TITLE
[IDEA] Re-Implement the 90x60 text mode

### ISSUE THREAD

**Issue by TheBigEye** [TheBigEye/Monarch-OS#7]
### Description
This seems more like an obsession than a necessity, but who understands me? Let's admit that a larger terminal is also more attractive and, in some cases, more efficient, right?

When using GRUB, something frustrating is that you can no longer switch to the 90x60 text mode with a simple interrupt, nor change the font size to 8x8 pixels, since we are talking about protected mode, a very difficult territory without the simplicity of the BIOS.

### So, what do we do?
- Still, it's not impossible. We can modify the text mode simply by changing the VGA CRTC registers to the correct values, so, in theory, we could have a 90x60 text mode.
- The problem lies more with the characters. With the VGA registers, you can change the size of the space they occupy on the screen, but not the size of the characters themselves. The only way to change the font size is by overwriting the memory offset where the BIOS-stored font bitmap is located with the desired bitmap. This is achieved by temporarily changing the way video memory is addressed (saving the current registers before doing so), enabling the last video plane, writing the font bytes to the correct address and offsets on that plane, and then restoring the registers. 
- In short, something like this: [https://wiki.osdev.org/VGA_Hardware#Sample_Register_Settings](https://wiki.osdev.org/VGA_Hardware#Sample_Register_Settings)
